### PR TITLE
[DOCFIX] Fix contribution task description

### PR DESCRIPTION
**Pull request by LuQQiu**
Add the description so that new contributors know only take one of the new contributor tasks.

### CODE REVIEW

**Comment by LuQQiu** [comment]
FYI @apc999 needs to create more new contributor tasks

**Comment by maobaolong** [comment]
@LuQQiu Thanks to maintain the new Contribution tasks, it helps us to bring newbie to alluxio open source community, and +1 to limit to take only one ticket for each new contributor.

**Review by maobaolong — APPROVED** [review]
As the `new-contributor-tasks` repo is reopened now, it would be better to have announcement  in slack, wechat, dingtalk or some other app to let more contributor candidate know this good news.

File: docs/en/contributor/Contributor-Getting-Started.md [hunk 1]
```diff
@@ -125,6 +125,8 @@ In order to assign an issue to yourself, leave a comment in the issue like `/ass
 to indicate that you are working on the issue.
 You should always assign a ticket to yourself this way before you start working on it, so others
 in the community know you are working on the ticket.
+The New Contributor issues are created for new contributors to try the whole contribution process,
```
[the remark this answers is not in the document]
Comment (LuQQiu) [reply docs/en/contributor/Contributor-Getting-Started.md]: ! Good catch! Changed, PTAL

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**